config/config.go

```diff
@@ -107,7 +113,7 @@ var baseConfig = Config{
 		MaxMemory:              0,
 		EvictionPolicy:         EvictAllKeysLFU,
 		EvictionRatio:          0.40,
-		KeysLimit:              10000,
+		KeysLimit:              20000000,
 		AOFFile:                "./dice-master.aof",
 		PersistenceEnabled:     true,
 		WriteAOFOnCleanup:      true,
```

Comment:
Let's disable AOF for now.
```suggestion
		WriteAOFOnCleanup:      false,
```